purchase_order_product_recommendation/wizards/purchase_order_recommendation.py

```diff
@@ -36,11 +36,16 @@ class PurchaseOrderRecommendation(models.TransientModel):
     )
     line_amount = fields.Integer(
         string='Number of recommendations',
-        default=0,
+        default=15,
         required=True,
         help='Stablish a limit on how many recommendations you want to get.'
              'Leave it as 0 to set no limit',
     )
+    show_all_partner_products = fields.Boolean(
+        string='Show all products',
+        default=False,
```

Comment:
@chienandalu no need to `default=False`

Reply:
@nikul-serpentcs remember about "explicit better than implicit", but any way, I think you should better focus your reviews about explicit guidelines or potential problems in the code, the UI, etc, as these nitpickings are not adding value to the review.